src/Measurement/Energy.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Sundata\Utilities\Measurement;
+
+class Energy implements Measurement
+{
+    private float $wattHour;
+    public int $precision = 2;
```

Comment:
precision is niet 'van het Measurement'. je kunt dezelfde measurement, bijvoorbeeld, meerdere keren naar een string omzetten met verschillende precisions. dus precision meegeven als arg op 't moment van format-en.